sale_order_type/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sale Order Types
+================
+
+This module adds a typology for the sale orders. In each different type, you
+can define, invoicing and refunding journal, a warehouse and a sequence.
+
+You are able to select a sale order type by partner so that when you add a
+partner to a sale order it will get the related type and warehouse to it.
+
+
```

Comment:
Add the section "Usage" (without text), but with the new cool button "Try on runbot":

```
.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/167/8.0
```